packages/api/src/routes/blog-subscribe.js

```diff
@@ -0,0 +1,86 @@
+import { JSONResponse } from '../utils/json-response.js'
+import btoa from 'btoa'
+
+const SERVER_PREFIX = 'us5'
+const LIST_ID = '64f6e3fd11'
+const API_KEY = process.env.MAILCHIMP_API_KEY
```

Comment:
CF will make this a global variable. Please deal with that and import from `constants.js`